Rename Valid language tag to Known primary language tag (#1738)
act-rules/act-rules.github.io: _rules/html-page-lang-matches-default-ucwvc8.md

This rule applies to any [document element][] if it is an `html` element for which all of the following are true:

- The [document element][] has a `lang` attribute with a value that has a [known primary language tag][]; and
- The [document element][] is in a [top-level browsing context][]; and
- The [document element][] has a [content type][] of `text/html`; and
- The [document element][] has a defined [default page language][].

## Expectation

> For each test target, the [primary language][] of the `lang` attribute matches the [default page language][] of the test target.

Reviewer: The new version is losing a bit the link to the tag 🤔 
```suggestion
For each test target, the [known primary language tag][] of its `lang` attribute matches the [default page language][] of the test target.
```

#### Inapplicable Example 5

> The `lang` [attribute value][] of this page is an [iso 639.2][] three letters code, hence not a [known primary language tag][].

Reviewer: ```suggestion
The `lang` [attribute value][] of this page is an [iso 639.2][] three letters code, hence has no [known primary language tag][].
```
(you may want to unify with phrasing used in other rules)

#### Inapplicable Example 6

> The `lang` [attribute value][] of this page is a [grandfathered tag][grandfathered tags], hence not a [known primary language tag][].

Reviewer: ```suggestion
The `lang` [attribute value][] of this page is a [grandfathered tag][grandfathered tags], hence has no [known primary language tag][].
```
